src/raw_decompiled/HEROES2W.h

```diff
@@ -3000,5 +2996,44 @@ struct SCmbtHero
 };
 #pragma pack(pop)
 
+/*  172 */
+#pragma pack(push, 1)
+struct sphinxMapExtra
```

Comment:
I suspect all these arrays in structures are examples of this: https://stackoverflow.com/a/36577185/546591

Reply:
This is just an FYI. I was making comments and scrolling up before I realized this was an automatically generated file.

Reply:
Ok. The flexible member thing is pretty cool, but I don't think I see how it can be reflected here. Perhaps I don't understand it well enough.

Reply:
What I mean is: These arrays are actually longer than one element. The structure is of variable size. If you ever move these definitions into code, you'll want to declare them without a fixed size.

Reply:
Ah, ok.